blog/models.py

```diff
@@ -22,6 +22,8 @@
 
 from company_website.models import Employees
 
+MAX_BLOG_ARTICLE_TITLE_LENGTH = 69
```

Comment:
Move it please to constants.py file

Reply:
There is an issue though: should I move this to _**company_website/constants.py**_? Because we have no constants defined for the blog app and this here is a blog-only value.
On the other hand, should we be creating a _**blog/constants.py**_ file at this point, when it would contain only a single constant?

Reply:
We should create **blog/constants.py**. This constant belongs to blog and there is it's place :)

Reply:
Done.